### PR TITLE
Docs/doc fixes for docs/maps/map-collection.md

### PR DESCRIPTION
**Pull request by goransle**

### CODE REVIEW

File: docs/maps/map-collection.md [hunk 1]
```diff
@@ -20,195 +20,83 @@ In the [map collection reference](http://code.highcharts.com/mapdata/), each map
 1. Add the map as a JavaScript element:
 
 ```html
-<script src="[http://code.highcharts.com/mapdata/custom/world.js](http://code.highcharts.com/mapdata/custom/world.js)"></script>
+<script src="http://code.highcharts.com/mapdata/custom/world.js"></script>
```
Comment (TorsteinHonsi) [inline docs/maps/map-collection.md]: Should be updated to `https://...`

File: docs/maps/map-collection.md [hunk 2]
```diff
@@ -20,195 +20,83 @@ In the [map collection reference](http://code.highcharts.com/mapdata/), each map
...
 ```
 
-You can alternatively link to a specific version or subversion of the map at http://code.highcharts.com/mapdata/**1.1**/custom/world.js. 
+<p>You can alternatively link to a specific version or subversion of the map at <code>http://code.highcharts.com/mapdata/<strong>1.1</strong>/custom/world.js</code>.</p>
```
Comment (TorsteinHonsi) [inline docs/maps/map-collection.md]: Should be `https://...`. Try doing a general search and replace in all the `/docs` for insecure URLs to our own services (`http://code.highcharts.com`, `http://api.highcharts.com`).

File: docs/advanced-chart-features/annotations-and-fibonacci-retracements.md [hunk 3]
```diff
@@ -123,4 +123,4 @@ Let’s create the Fibonacci retracements chart with seven labels, seven horizon
         };
     
 
-Check the live demo [here](http://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/annotations/fibonacci-retracements).
+Check the live demo [here](https://jsfiddlefiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/annotations/fibonacci-retracements).
```
Comment (TorsteinHonsi) [inline docs/advanced-chart-features/annotations-and-fibonacci-retracements.md]: Ouch, something went wrong with the text replacement with all the jsFiddle linkes  (`http://jsfiddle.net` => `https://jsfiddlefiddle.net`).

Comment (goransle) [reply docs/advanced-chart-features/annotations-and-fibonacci-retracements.md]: Should be corrected now.

**Pull request merged**